chart-parts/ingress.yaml

```diff
@@ -1,40 +1,39 @@
-{{- if .Values.services.ingress -}}
+{{- if .Values.ingress.enabled -}}
```

Comment:
Do we have any backwards-compatibility concerns here?

Reply:
No.